src/controllers/request.controller.js

```diff
@@ -0,0 +1,39 @@
+import RequestService from '../services/request.service';
+import ResponseService from '../services/response.service';
+
+/**
+ *
+ *
+ * @class RequestController
+ */
+class RequestController {
+  /**
+   *
+   *
+   * @static
+   * @param {req} req
+   * @param {res} res
+   * @returns {response} @memberof RequestController
+   */
+  static async rejectRequest(req, res) {
+    const [, [{ dataValues }]] = await RequestService.updateRequest({ id: req.params.requestId }, { status: 'rejected' });
+    ResponseService.setSuccess(200, 'Request has successfully rejected', dataValues);
+    return ResponseService.send(res);
+  }
+
+  /**
+   *
+   *
+   * @static
+   * @param {req} req
+   * @param {res} res
+   * @returns {response} @memberof RequestController
+   */
+  static async approveRequest(req, res) {
```

Comment:
Instead of having different controllers to handle approval/rejection of requests, you can get the status from the request payload and set that instead.